src/Tooltips/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import ReactTooltip from 'react-tooltip'
+import * as S from './styles'
+import { Props } from "./types"
+
+export const Tooltip = ({ effect, place, tip, children, type }: Props) => {
+    /**
+     * @description the ReactTooltip Component need a different id
+     */
+
+    const id = Math.floor((1 + Math.random()) * 0x10000).toString(16).substring(1) + tip
+    return (
+        <S.Container data-testid="container">
+            <div data-testid='tooltip'>
+                <ReactTooltip effect={effect?.length ? effect : 'solid'} place={place?.length ? place : 'top'} id={id} border />
```

Comment:
Tanto `effect` e `place` são strings, portanto, é melhor atribuir um valor padrão nos props do que verificar se o comprimento da string é maior que 0.
Ex: 
```tsx
const Component = ({effect = 'solid', place = 'top'}: Props) => <div />
```

E assim você consegue remover o ternário daqui

```suggestion
                <ReactTooltip effect={effect} place={place} id={id} border />
```

Se seguir essa sugestão, inclua na documentação também o valor padrão de cada prop.